webpack.docs.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
     },
   },
   plugins: [
-    new webpack.HotModuleReplacementPlugin()
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoErrorsPlugin(),
```

Comment:
This extra plugin did not solve the issue with the JSX parer stopping the build.  Let's pull it since it is doing nothing.